<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>theme</class>
 <widget class="QMainWindow" name="theme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="whatsThis">
   <string>theme-box</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_7" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1070</width>
      <height>650</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>theme-mainbox</string>
    </property>
    <widget class="QWidget" name="current_theme" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>220</width>
       <height>650</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>theme-operatebox</string>
     </property>
     <widget class="QWidget" name="widget_white" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>200</width>
        <height>540</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>singel-box-white</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>180</width>
         <height>480</height>
        </rect>
       </property>
       <property name="whatsThis">
        <string>white-item</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background:rgb(255,255,255);
border:1px solid #000;
border-radius:10px;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>500</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <property name="whatsThis">
        <string>white-label</string>
       </property>
       <property name="text">
        <string>淡雅白</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_singal" native="true">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>200</width>
       <height>540</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>singal-box-gray</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>180</width>
        <height>480</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>gray-item</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>500</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>gray-label</string>
      </property>
      <property name="text">
       <string>冷漠灰</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_diy" native="true">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>10</y>
       <width>200</width>
       <height>540</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>singal-box-diy</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>180</width>
        <height>480</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>150</pointsize>
       </font>
      </property>
      <property name="whatsThis">
       <string>diy-item</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>?</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>500</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>diy-label</string>
      </property>
      <property name="text">
       <string>自定义</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>570</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>btn</string>
     </property>
     <property name="text">
      <string>退出</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>10</y>
       <width>200</width>
       <height>540</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>singal-box-blue</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>180</width>
        <height>480</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>blue-item</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>500</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>blue-label</string>
      </property>
      <property name="text">
       <string>天空蓝</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>570</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>btn</string>
     </property>
     <property name="text">
      <string>配置</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_pink" native="true">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>200</width>
       <height>540</height>
      </rect>
     </property>
     <property name="whatsThis">
      <string>singal-box-pink</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>180</width>
        <height>480</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>pink-item</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>500</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>pink-label</string>
      </property>
      <property name="text">
       <string>甜心粉</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
